pkg/service/service.go

```diff
@@ -851,8 +851,10 @@ func (s *Service) restoreBackendsLocked() error {
 			logfields.L3n4Addr:  b.L3n4Addr.String(),
 		}).Debug("Restoring backend")
 		if err := RestoreBackendID(b.L3n4Addr, b.ID); err != nil {
-			return fmt.Errorf("Unable to restore backend ID %d for %q: %s",
-				b.ID, b.L3n4Addr, err)
+			log.WithError(err).WithField(logfields.BackendID, b.ID).
+				Warnf("Unable to restore backend ID %d for %q: %s",
+					b.ID, b.L3n4Addr, err)
+			continue
```

Comment:
How about putting all arguments as fields and removing them from the message?

```
log.WithFields(logrus.Fields{
                logfields.BackendID: b.ID,
		logfields.L3n4Addr: b.L3n4Addr,
	}).Warn("Unable to restore backend")
```

Reply:
done